assets/scss/pages/login-page.scss

```diff
@@ -0,0 +1,168 @@
+.c-login__section {
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    gap: 1.5rem;
+
+    @include bp(medium) {
+        padding: 3rem 5rem;
+        gap: 3rem;
+    }
+}
```

Comment:
@mgbelegu same as contact page. I'd suggest `p-login` as a main element on the page. Adjust children elements accordingly.